### ISSUE TITLE
hdfs commands via hadoop deprecated?

### ISSUE THREAD

**Issue by PKRoma** [alexholmes/hiped2#1]
If it is true that the use of the generic hadoop command is being deprecated (edit the hadoop command for comments) in favour of the specialised hdfs commands etc., maybe the examples should reflect that? E.g., in the README.md

hadoop fs -mkdir -p hip1/input

should become

hdfs dfs -mkdir -p hip1/input

**Comment by krinkere** [comment]
Perhaps a better way is to create your own repo with all those dependencies so that people can just download it from you and you won't have to edit pom.xml file?

**Comment by PKRoma** [comment]
Yes, that is one way of doing it, but given I'm a Hadoop newbie, this was more of a question of guidance by someone more expert than me: Should I be learning and using direct 'hdfs' commands instead of 'hadoop fs'. I wouldn't want to change everything that way in the examples before seeing opinions from people more expert than me. And, changing the examples may make them diverge from the text in the book...
